fdbserver/Status.actor.cpp

```diff
@@ -839,7 +839,9 @@ ACTOR static Future<JsonBuilderObject> processStatusFetcher(
 	return processMap;
 }
 
-static JsonBuilderObject clientStatusFetcher(ClientVersionMap clientVersionMap, std::map<NetworkAddress, std::string> traceLogGroupMap) {
+static JsonBuilderObject clientStatusFetcher(ClientVersionMap clientVersionMap,
+									std::map<NetworkAddress, std::string> traceLogGroupMap,
```

Comment:
combine the logGroup and the TLS map into one map. the second parameter is a structure.